tox.ini

```diff
@@ -0,0 +1,52 @@
+[tox]
+envlist = linters
+skipsdist = true
+
+[testenv]
+allowlist_externals=*
+deps =
+       -r{toxinidir}/test_requirements.txt
+commands =
+    {envpython} --version
+
+[testenv:venv]
+basepython = python3
+commands =
+    {posargs}
+
+[testenv:install]
+basepython = python3
+allowlist_externals =
+    bash
+commands =
+    bash -c "echo '==> Installing Event Driven Ansible...'"
+    bash -c "ansible-galaxy collection build -v --force --output-path releases/ && \
+             ansible-galaxy collection install --force --force-with-deps releases/ansible-eda-$(cat ./galaxy.yml | shyaml get-value version).tar.gz"
+
+[testenv:ruff]
+deps = {[testenv:linters]deps}
+commands =
+    bash -c 'ruff check --select ALL --ignore INP001 extensions/eda/plugins || true'
```

Comment:
The idea here is to push additional PRs removing the || true once each lint check is passing.